app/v1/controllers/requestParsers/validators/validations/TaxYearValidation.scala

```diff
@@ -16,32 +16,31 @@
 
 package v1.controllers.requestParsers.validators.validations
 
-import v1.models.errors.{MtdError, RuleTaxYearRangeInvalidError, TaxYearFormatError}
+import v1.models.errors.{MtdError, RuleTaxYearRangeInvalid, TaxYearFormatError}
 
 object TaxYearValidation {
 
   val taxYearFormat = "20[1-9][0-9]\\-[1-9][0-9]"
 
   def validate(taxYear: String): List[MtdError] = {
-    if (taxYear.matches(taxYearFormat)) {
-
-      val startTaxYearStart: Int = 2
-      val startTaxYearEnd: Int = 4
-
-      val endTaxYearStart: Int = 5
-      val endTaxYearEnd: Int = 7
-
-      val start = taxYear.substring(startTaxYearStart, startTaxYearEnd).toInt
-      val end   = taxYear.substring(endTaxYearStart, endTaxYearEnd).toInt
-
-      if (end - start == 1) {
-        NoValidationErrors
+    try {
+      if (taxYear.matches(taxYearFormat)) {
+
+        //scalastyle:off
+        val start = taxYear.substring(2, 4).toInt
```

Comment:
use of variable names makes it easier to understand what the 2, 4, 5 and 7 bits mean.

Reply:
Yep, I'll revert back to that